FrontEnd/src/pages/Auth.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useAuthStore } from "../store/authStore";
+import LoginForm from "../components/LoginForm";
+import SignupForm from "../components/SignUpForm";
+
+export default function Login() {
```

Comment:
```suggestion
export default function Auth() {
```
Component name should match the file name, and it's a little bit confusing to have `Login` components that holds the ` SignupForm`. If we faced a bug on signup, the last component we will check is the Login component.